oomcli/test/test_update_group.sh

```diff
@@ -9,11 +9,9 @@ import_sample > /dev/null
 case='oomcli update group works'
 oomcli update group phone --description "new description"
 expected='
-Name:             phone
-Entity:           device
-Description:      new description
-OnlineRevisionID: <NULL>
+NAME,ENTITY,DESCRIPTION
+phone,device,new description
 '
-actual=$(oomcli describe group phone)
-ignore() { grep -Ev '^(CreateTime|ModifyTime)' <<<"$1"; }
+ignore() { cut -d ',' -f 2-4 <<<"$1"; }
+actual=$(oomcli get meta group phone -o csv)
```

Comment:
Suggest list complete output in `expected` and do the same filter logic on `expected` and `actual` so we can known what the output looks like and what was filtered actually.

Reply:
sounds good. updated.